### PR TITLE
Fix lint and requirements

### PR DESCRIPTION
**Pull request by cawallin**

### CODE REVIEW

**Comment by cawallin** [comment]
@kokosing @rschlussel

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 import os
 
 # This is necessary for nose to handle multiprocessing correctly
-from multiprocessing import util
+from multiprocessing import util  # noqa
```
Comment (rschlussel) [inline setup.py]: what is the flake8 error that would get raised here?

Comment (cawallin) [inline setup.py]: unused import, but I wasn't sure about removing it b/c of the comment

**Comment by rschlussel** [comment]
looks good.

**Comment by kokosing** [comment]
looks good

Maybe you could make line length limit longer just for convenience. Now you had to split so many lines, with longer line length limit you could avoid that.

**Comment by cawallin** [comment]
We were hesitant to change the line length limit, since that's the standard PEP-8 limit. Might as well keep it for now.

**Pull request merged**